src/amo/components/Icon/styles.scss

```diff
@@ -290,33 +249,10 @@
   @include permission-icon($name: 'navigation');
 }
 
-.Icon-developer {
-  @include icon($name: 'developer');
-}
```

Comment:
We need this one in the user profile of a developer, e.g., https://addons-dev.allizom.org/en-US/firefox/user/10641750/

Reply:
I'm sorry. I will revert it.